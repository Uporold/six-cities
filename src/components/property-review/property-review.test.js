import React from "react";
import renderer from "react-test-renderer";
import PropertyReview from "./property-review";

const review = {
  id: 1,
  user: {
    id: 14,
    isPro: true,
    name: "Corey",
    avatar:
      "https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/5.jpg",
  },
  rating: 4,
  comment:
    "Home is amazing. It's like staying in a museum. The rooms, furnishings and artworks are incredible. The views of My Vesuvius",
  date: "2020-07-29T07:00:10.725Z",
};

it(`Should Reviews List render correctly`, () => {
  const tree = renderer.create(<PropertyReview review={review} />).toJSON();

  expect(tree).toMatchSnapshot();
});
