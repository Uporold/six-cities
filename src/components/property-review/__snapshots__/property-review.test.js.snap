// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Reviews List render correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper  property__avatar-wrapper--pro"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="https://htmlacademy-react-3.appspot.com/six-cities/static/avatar/5.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Corey
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Home is amazing. It's like staying in a museum. The rooms, furnishings and artworks are incredible. The views of My Vesuvius
    </p>
    <time
      className="reviews__time"
      dateTime="2020-07-29T07:00:10.725Z"
    >
      July 2020
    </time>
  </div>
</li>
`;
